runtime/lua/vim/clipboard/osc52.lua

```diff
@@ -2,23 +2,23 @@ local M = {}
 
 function M.copy(lines)
   local s = table.concat(lines, '\n')
-  io.stdout:write(string.format('\x1b]52;;%s\x1b\\', vim.base64.encode(s)))
+  io.stdout:write(string.format('\027]52;;%s\027\\', vim.base64.encode(s)))
```

Comment:
We need a lint rule for this. https://github.com/lunarmodules/luacheck/issues/37#issuecomment-977086045 mentions a way to extend luacheck. But maybe treesitter is a better way for us to maintain Lua lint rules.